peps/pep-0731.rst

```diff
@@ -0,0 +1,181 @@
+PEP: 731
+Title: C API Working Group Charter
+Author: Guido van Rossum <guido@python.org>,
+        Petr Viktorin <encukou@gmail.com>,
+        Victor Stinner <vstinner@python.org>,
+        Steve Dower <steve.dower@python.org>,
+        Irit Katriel <irit@python.org>
+Status: Draft
+Type: Process
+Topic: Typing
```

Comment:
```suggestion
```

Possibly there should be a Topic for the C API, but this is definitely not about typing.